test/test_av.py

```diff
@@ -86,7 +87,7 @@ def test_tag_cells(ctx_factory, dim, order):
     queue = cl.CommandQueue(cl_ctx)
     actx = PyOpenCLArrayContext(queue)
 
-    nel_1d = 2
+    nel_1d = 1
```

Comment:
🤔 Not complaining, just curious.

Reply:
The old `generate_regular_rect_mesh` argument `n` specified the number of mesh vertices along a given axis, so the test was creating a mesh with 2 vertices, or 1 element. Now it does the same thing but with `nelements_per_axis`.

Not sure whether the test was originally intended to have 1 or 2 elements. At the moment it fails if I try to use a mesh with more than 1 element (whether passing `n` or `nelements_per_axis`).